### PR TITLE
Add getChainTips rpc

### PR DESCRIPTION
**Pull request by jingyi2811**
/kind feature

Add getChainTips rpc

### CODE REVIEW

File: website/docs/jellyfish/api/blockchain.md [hunk 1]
```diff
@@ -171,7 +171,18 @@ interface blockchain {
...
+
+Return information about all known tips in the block tree
+including the main chain as well as orphaned branches.
+
+```ts title="client.blockchain.getChainTips()"
+interface blockchain {
+  getChainTips (): Promise<ChainTip[]>
+}
+```
+
```
Comment (fuxingloh) [inline website/docs/jellyfish/api/blockchain.md:184]: where is your `ChainTip` interface?

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: website/docs/jellyfish/api/blockchain.md [hunk 2]
```diff
@@ -171,7 +171,25 @@ interface blockchain {
...
+interface ChainTip {
+  height: number
+  hash: string
+  branchlen: number
+  status: string
+}
+```
+
+# getTxOut
```
Comment (fuxingloh) [inline website/docs/jellyfish/api/blockchain.md:192]: `## getTxOut`

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/blockchain.test.ts [hunk 3]
```diff
@@ -235,6 +235,26 @@ describe('masternode', () => {
...
+        expect(data.height).toBeGreaterThan(0)
+        expect(typeof data.hash).toBe('string')
+        expect(data.hash.length).toBe(64)
+        expect(data.branchlen).toBeGreaterThanOrEqual(0)
+        expect(
+          data.status === 'invalid' ||
+          data.status === 'headers-only' ||
+          data.status === 'valid-headers' ||
+          data.status === 'valid-fork' ||
+          data.status === 'active'
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/blockchain.test.ts:252]: use `.includes` to be cleaner

**Review by canonbrother — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/blockchain.test.ts (hunk 3 above)
Comment (jingyi2811) [reply packages/jellyfish-api-core/__tests__/category/blockchain.test.ts:252]: Likewise, .includes has the same problem like .toContain

Fuxing: It must always be like this, testing is like reading a narrative.
expect (actual) to be (expected)

Comment (fuxingloh) [reply packages/jellyfish-api-core/__tests__/category/blockchain.test.ts:252]:
```ts
expect(['invalid', 'headers-only', ...].includes(data.status)).toBe(true)
```

Comment (jingyi2811) [reply packages/jellyfish-api-core/__tests__/category/blockchain.test.ts:252]: Seems similar to the code I wrote earlier.

`expect(['invalid', 'headers-only', 'valid-headers', 'valid-fork', 'active']).toContain(data.status)`

I followed what was taught here https://jestjs.io/docs/expect#tocontainitem

Comment (jingyi2811) [reply packages/jellyfish-api-core/__tests__/category/blockchain.test.ts:252]: Okay I shall use this -> expect(['invalid', 'headers-only', ...].includes(data.status)).toBe(true). Will commit and push before end of today

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**